contentcuration/contentcuration/frontend/shared/data/__tests__/ContentNodeResource.spec.js

```diff
@@ -1,5 +1,100 @@
 import sortBy from 'lodash/sortBy';
-import { ContentNode, ContentNodePrerequisite } from 'shared/data/resources';
+import {
+  RELATIVE_TREE_POSITIONS,
+  COPYING_FLAG,
+  TASK_ID,
+  CHANGES_TABLE,
+  CHANGE_TYPES,
+} from 'shared/data/constants';
+import db, { CLIENTID } from 'shared/data/db';
+import { ContentNode, ContentNodePrerequisite, uuid4 } from 'shared/data/resources';
+
+describe('ContentNode methods', () => {
+  const mocks = [];
+  afterEach(() => {
+    mocks.forEach(mock => mock.mockRestore());
+    return ContentNode.table.clear();
+  });
+
+  function mockContentNodeFunc(name, implementation) {
+    const mock = jest.spyOn(ContentNode, name).mockImplementation(implementation);
+    mocks.push(mock);
+    return mock;
+  }
+
+  describe('tableCopy method', () => {
+    it('should load relevant nodes', () => {
+      const nodes = ['Copy node', 'Target node'].map(title => ({
+        id: uuid4(),
+        channel_id: uuid4(),
+        root_id: uuid4(),
+        parent: uuid4(),
+        source_node_id: uuid4(),
+        original_source_node_id: uuid4(),
+        node_id: uuid4(),
+        title,
+      }));
+      const [copyNode, targetNode] = nodes;
+
+      let getNewParentAndSiblings = mockContentNodeFunc('getNewParentAndSiblings', () => {
+        return Promise.resolve({ parent: targetNode.id, siblings: [] });
+      });
+
+      let requestModel = mockContentNodeFunc('requestModel', id => {
+        return Promise.resolve(nodes.find(n => n.id === id));
+      });
+
+      const excluded_descendants = 'excluded_descendants';
```

Comment:
Don't think it makes a difference to the test, but this argument is meant to be an object, so might be nice to use one here.